sale_order_customer_free_ref/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+{
+    "name": "Sale Order Customer Free Reference",
+    "version": "13.0.1.0.0",
+    "category": "Sale",
+    "license": "AGPL-3",
+    "summary": "Splits the Customer Reference on sale orders into two fields."
+    "An Id and a Free reference. The existing field is transformed"
+    "into a computed one.",
```

Comment:
Those strings will be concatenated so no space will be left between words being in the tails of each line.

```suggestion
    "summary": "Splits the Customer Reference on sale orders into two fields. "
    "An Id and a Free reference. The existing field is transformed "
    "into a computed one.",
```

Reply:
Thanks fixed.